### DISCUSSION TITLE
find help for robo-gym

### DISCUSSION THREAD

**Discussion by Plusmile** [jr-robotics/robo-gym#69] [Q&A]
dear matteo:
I'm going to work on UR robotic grasping, which uses reinforcement learning.
After some investigation, I found robo-gym and it really worked for me, I also looked at the robo-gym paper.
I want to complete the task of "End Effector Positioning" first, which is to reproduce what you mentioned in your paper. My current idea is to pull the robo-gym-robot-server-side image on dockerhub, and then generate more than a dozen container to generate training data and  then trained using a reinforcement learning algorithm.
There is something I don't understand about this idea.
1) How to use server_manager to manage these dozen or so containers, does the server_manager in the container need to be started? Or do I just need to start the server_manager in the host?
2) env = gym.make('', ip='<server_manager_address>'), the ip in this statement refers to the ip in the container or the ip on the host
3) I changed the ip to the container ip, and after starting the server_manager of the host, the program did not report an error, but the robot_server of the container was not working when using the TOP command to monitor, but the robot_server of the host was running, why is this?
I'm really looking forward to your guidance, my mentors can be of limited help and I'm feeling lost right now.

**Comment by Adarsh5harma** [comment]
Hii @Plusmile ,

Did you managed to get it worked with real robot?

**Reply by Plusmile** [reply]
yes,on UR3,my email is plusmile@qq.com.We can help each other

**Reply by Adarsh5harma** [reply]
Hii, yes I wrote you an email.

**Reply by Adarsh5harma** [reply]
Hey @Plusmile, can you help me out? I wrote you an email regarding my issue.
